tensorboard/plugins/projector/projector_plugin_test.py

```diff
@@ -56,7 +57,10 @@ def testRunsWithValidCheckpoint(self):
     self._GenerateProjectorTestData()
     self._SetupWSGIApp()
     run_json = self._GetJson('/data/plugin/projector/runs')
-    self.assertTrue(run_json)
+    if tf_compat.__version__ != 'stub':
```

Comment:
I don't think most of these tests are very meaningful when running under the stub TF (some of the ones below just don't even do any asserts in that case, at which point performing the setup anyway is wasteful).

Instead of using if statements like this, can you just create a boolean at the top for e.g. `USING_REAL_TF = tf_compat.__version__ != 'stub'` and then decorate all the affected tests as follows:

```python
# TODO(#2007): Cleanly separate out projector tests that require real TF
@unittest.skipUnless(USING_REAL_TF, 'Test only passes when using real TF')
def testSomething():
  ...
```

Reply:
My bad on the tests that are purely setup - I'll fix those. However, there are a number that are valid and I'll use the approach you outlined above. Thanks.